iris_doc/api_tagger.py

```diff
@@ -495,7 +535,7 @@ def tokenize(self) -> List[Token]:
             extensionName = self.__syntaxMatcher.matchExtension(fileLine)
             if extensionName:
                 extensionTokens = self._getClassTokens(
-                    className=extensionName, lineIndex=lineIndex, type=TYPE_EXTENSION)
+                    className=extensionName.replace('(','').replace(')',''), lineIndex=lineIndex, type=TYPE_EXTENSION)
```

Comment:
Is that mean the `extensionName` may contain `(`/`)`?

Reply:
Yep, with iOS/macOS it gets the extension name as AgoraRtcEngine(Ex), but the class is actually AgoraRtcEngineEx
<img width="486" alt="Screenshot 2023-04-25 at 11 54 31" src="https://user-images.githubusercontent.com/5754073/234255881-b2d3c8a8-606d-41fa-8886-079038d028ef.png">

Reply:
I think the extension name should be `AgoraRtcEngineKit(Ex)` according to the code.

```objective-c
NS_ASSUME_NONNULL_BEGIN
@interface AgoraRtcEngineKit(Ex)
...
@end
```

Reply:
Looks like it's okay without, I have removed these parts.